.prettierignore

```diff
@@ -15,5 +15,4 @@ packages/amplify-e2e-tests/amplify-e2e-reports
 packages/amplify-graphql-model-transformer/publish-notification-lambda/lib/
 packages/amplify-graphql-model-transformer/rds-lambda/
 packages/amplify-graphql-model-transformer/rds-patching-lambda/lib/
```

Comment:
We wouldn't need any of these for Gen 1 right?

Reply:
We don't need them in Gen-1, but prefer to keep them until we migrate the set of E2Es we run on `main` that use the Gen-1 plugin.